xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd

```diff
@@ -368,6 +373,52 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether  travelling with Pet.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="reviews" type="reviews_RelStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Reviews associated to the passenger</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
 		</xsd:sequence>
 	</xsd:group>
+	<!-- ====  REVIEWS.  ============================================ -->
+	<xsd:element name="Review" type="ReviewStructure">
+		<xsd:annotation>
+			<xsd:documentation>A review of a diver or of a traveler.</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="reviews_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for collection of reviews.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element ref="Review" maxOccurs="unbounded"/>
+		</xsd:sequence>
+	</xsd:complexType>
+	<xsd:complexType name="ReviewStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for Review.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Date" type="xsd:dateTime" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Date and time of the review</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="Ranking" type="xsd:positiveInteger">
```

Comment:
Add an explicit range.

Reply:
That's in the annotation below (_most often 1 to 5, and the higher is the better_) . I'm not sure if we can put a constraint on it (there are other ranking mechanisms than the 5 stars), and other Rankings in NeTEx are totally free.

Reply:
Thinking more in this direction.
```xsd
  <restriction base='integer'>
    <minInclusive value='1'/>
    <maxInclusive value='5'/>
  </restriction>
```

Reply:
Yes, that can be nice ... but to be agreed by the group and decide if we have to generalise to other Rankings (also for the global ranking some services have a decimal value being the mean of individual rankings).